pkg/composer/logs.go

```diff
@@ -39,7 +40,7 @@ type LogsOptions struct {
 	NoLogPrefix bool
 }
 
-func (c *Composer) Logs(ctx context.Context, lo LogsOptions, services []string) error {
+func (c *Composer) Logs(ctx context.Context, lo LogsOptions, services []string, abortOnContainerExit bool) error {
```

Comment:
Why not add this to `LogsOption`?

Reply:
I thought it was out of scope, but if it's ok for you I can move it under the `LogsOption` struct